src/llama-graph.cpp

```diff
@@ -1031,11 +1042,12 @@ ggml_tensor * llm_graph_context::build_inp_pos() const {
 }
 
 ggml_tensor * llm_graph_context::build_inp_attn_scale() const {
-    auto inp = std::make_unique<llm_graph_input_attn_temp>(n_pos_per_token(), hparams.n_attn_temp_floor_scale, hparams.f_attn_temp_scale);
+    auto inp = std::make_unique<llm_graph_input_attn_temp>(n_pos_per_embd(), hparams.n_attn_temp_floor_scale, hparams.f_attn_temp_scale);
```

Comment:
@ggerganov Because `build_inp_attn_scale` is currently used exclusively by llama 4, do you think we should get rid of `n_pos_per_embd` and replace it with a `GGML_ASSERT(n_pos_per_embd() == 1)` ?

Reply:
Yes, we can do that.

Reply:
On second thought, `build_inp_attn_scale` should work well even in the case of N pos per token.

That's because the scale is applied per embedding, and the number of embedding is independent from N pos per token.

In any cases, I removed the `n_pos_per_embd` in https://github.com/ggml-org/llama.cpp/pull/13138/commits/9cd16a3981cb96ed7d69df9121e88288358b83cb , merging this PR once the CI is green